aci/convert/convert.go

```diff
@@ -110,7 +111,9 @@ func ToContainerGroup(ctx context.Context, aciContext store.AciContext, p types.
 			dnsLabelName = serviceDomainName
 		}
 
-		containers = append(containers, containerDefinition)
+		containerDefinition.LivenessProbe = service.getLivenessProbe()
```

Comment:
move the call of getLivenessProbe() into getAciContainer() (called above, L93)

Reply:
Totally!